.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Let's leave editor configs out of the source tree. Please add this to the `.gitignore` file

Reply:
`.editorconfig` is a universal basic config file recognized by a lot of editors / ides

Reply:
TIL

I thought it was Atom-specific(after doing a basic google search). I can see how it might be useful then!

Reply:
I can check in webstorm's config dir if you want it :p